index.html

```diff
@@ -0,0 +1,228 @@
+      <div class="container">
+        <div class="row">
+          <div class="col-6">
+            <h3 class="travel-text__submiddle">BHROMAON</h3>
+          </div>
+          <div class="col-5">
+            <div class="nav">
+              <nav class="desctop-nav">
+                <ul class="desctop-nav__list">
+                  <li class="desctop-nav__item">
+                    <a href="#" class="menu__link">Who We Are</a>
+                  </li>
+                  <li class="desctop-nav__item">
+                    <a href="#" class="menu__link">What We Do</a>
+                  </li>
+                  <li class="desctop-nav__item">
+                    <a href="#" class="menu__link">Travel With Us</a>
+                  </li>
+                  <li class="desctop-nav__item">
+                    <a href="#" class="menu__link">Contact Us</a>
+                  </li>
+                </ul>
+              </nav>
+            </div>
+          </div>
+          <div class="col-1">
+            <button class="burger"><span></span></button>
+          </div>
+        </div>
+        <nav class="mobile-nav">
+          <ul class="mobile-nav__list">
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">Who We Are</a>
+            </li>
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">What We Do</a>
+            </li>
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">Travel With Us</a>
+            </li>
+            <li class="mobile-nav__item">
+              <a href="#" class="mobile-nav__link">Contact Us</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </div>
+    <div class="travel__why">
+      <div class="container">
+        <div class="row">
+          <div class="col-6">
+            <div class="container__why">
+              <h2 class="travel-text__bigger">Why Choose Us</h2>
+              <h4 class="travel-text__small">
+                Tion and dislike men who are so beguiled and demoralized by
+                desire by the charms of pleasure of the moment, so blinded by
+                desire, that they cannot foresee Ddislike men who are so
+                beguiled and demoralized by the charms of pleasure of the
+                moment, so blinded by desire, that.
+              </h4>
+              <button class="travel-button__explores">Eplore Places</button>
+            </div>
+          </div>
+          <div class="col-5">
+            <img src="img/Image1.png" alt="" />
+            <img class="margeningtotheright" src="img/Image2.png" alt="" />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="travel__steps">
+      <div class="container">
+        <div class="row">
+          <div class="col-4">
+            <img src="img/search.png" alt="" />
+            <h2 class="travel-bigger">Search Places</h2>
```

Comment:
лучше завернуть ещё в блок
```suggestion
          <div class="step">
              <img src="img/search.png" class="step__icon" alt="search icon" />
              <h2 class="step__title travel-bigger">Search Places</h2>
            </div>
```